packages/showcases/css/stories/components/indicators/progressbar/examples/overview.html

```diff
@@ -1,51 +1,54 @@
-<div class="block">
-  <div
-    class="vtmn-progressbar_container"
-    role="progressbar"
-    aria-labelledby="progress-bar-overview-1"
-    aria-valuemin="0"
-    aria-valuemax="100"
-    aria-valuenow="60"
-  >
-    <div class="vtmn-progressbar_label">
-      <span id="progress-bar-overview-1">Loading</span>
-      <span aria-live="assertive">60%</span>
+<div style="width: 300px">
+  <div class="block">
+    <div
+      class="vtmn-progressbar_container vtmn-progressbar_variant--linear"
```

Comment:
In the overview examples I used to be as compact as possible wit h the components. That's why sometimes you don't have the vtmn-progressbar_variant-- or vtmn-progressbar_size-- classes to display the default css styles. So for this example I would remove it too.
```suggestion
      class="vtmn-progressbar_container"
```